container-build-task/Test/gcloud-fails.ts

```diff
@@ -17,14 +17,12 @@
  *   a failure in gcloud.
  * @author przybjw@google.com (Jim Przybylinski)
  */
+import {getDefaultAnswers, registerCommonMocks} from 'common/register-mocks';
 import * as path from 'path';
-import * as fs from 'fs';
-import { TaskLibAnswers } from 'vsts-task-lib/mock-answer';
-import { TaskMockRunner } from 'vsts-task-lib/mock-run';
-import {
-  registerCommonMocks, getDefaultAnswers, getFsMock
-} from 'common/register-mocks';
-import { failResult } from './test-constants'
+import {TaskLibAnswers} from 'vsts-task-lib/mock-answer';
+import {TaskMockRunner} from 'vsts-task-lib/mock-run';
+
```

Comment:
nit: Should we remove the line here? Same for:
container-build-task/Test/using-custom-config.ts
container-build-task/Test/using-default-config-all-params.ts
container-build-task/Test/using-default-config.ts
container-build-task/Test/using-dockerfile-all-params.ts
container-build-task/Test/using-dockerfile.ts